ui/app/pages/swaps/dropdown-search-list/index.scss

```diff
@@ -1,7 +1,5 @@
 .dropdown-search-list {
   flex-flow: column;
```

Comment:
Did you find out what this rule was intended to do after?

Reply:
Removed thanks to Firefox!